pkg/database/migrations.go

```diff
@@ -31,6 +31,8 @@ const initState = "00000-Init"
 func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 	logger := logging.FromContext(ctx)
 	options := gormigrate.DefaultOptions
+	options.UseTransaction = true
```

Comment:
I think this will break some of the older migrations

Reply:
I re-ran all of them a few times from scratch - only one was broken and it's fixed later.